package.json

```diff
@@ -68,7 +68,6 @@
     "**/@typescript-eslint/eslint-plugin": "^6.7.3",
     "**/@typescript-eslint/parser": "^6.7.3",
     "webpack": "4.44.2",
-    "react-error-overlay": "6.0.9",
```

Comment:
just removing an old dep that we don't need anymore that was causing some issues on netlify